covid_symptoms/Dockerfile

```diff
@@ -1,34 +1,31 @@
-FROM openjdk:8-alpine
+FROM eclipse-temurin:8-jdk
 
 # Set to your proxies if necessary to build behind firewalls:
 #ENV HTTP_PROXY="http://10.41.13.6:3128"
 #ENV HTTPS_PROXY="http://10.41.13.6:3128"
 #ENV FTP_PROXY="http://10.41.13.6:3128"
 
-RUN apk update && apk add ca-certificates openssl wget unzip subversion maven
-RUN apk upgrade
+RUN apt-get update && apt-get install -y ca-certificates openssl wget unzip subversion maven
 
-RUN apk add --no-cache nss
-## Download apache-tomcat and extract:
-RUN wget -q https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.21/bin/apache-tomcat-9.0.21.zip
-RUN unzip -q apache-tomcat-9.0.21.zip
+RUN apt-get install -y libnss3
```

Comment:
Could go in above `apt-get` line?

Reply:
probably - this is a holdover from this originally being apk with no-cache